modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -71,6 +71,14 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 {productname} 6.5 also includes the following bug fixes:
 
+=== On iOS Safari, Hangul (Korean) characters will no longer merge onto the previous line after inserting a newline by pressing Enter.
```

Comment:
As I found out in my follow-up PR for this ticket, Hangul is only a category of Korean characters. I've modified that changelog entry already but this must have been transferred over before that happened.
```suggestion
=== On iOS Safari, Korean characters will no longer merge onto the previous line after inserting a newline by pressing Enter.
```

Reply:
Sounds good, ill update the incoming changes to the changelog for 6.5